dev-tools/deploy/preflight.py

```
@@ -285,23 +293,140 @@ def is_missing(value: Any) -> bool:
     for participant, result in value.items():
-        if not isinstance(result, dict) or result.get("disposition") not in safe_dispositions:
+        disposition = result.get("disposition") if isinstance(result, dict) else None
+        if disposition not in SAFE_RECOVERY_DISPOSITIONS:
             return (
                 "fail",
                 f"Recovery compatibility baseline {label} has unsafe or missing disposition: {participant}",
             )
     return ("pass", "")
 
 
+def validate_erasure_overlay_reconciliation(
```

Ruff (0.16.0)

[warning] Too many return statements (21 > 6)

(PLR0911)